runtime/src/consensus/tendermint/verifier.rs

```diff
@@ -121,6 +122,7 @@ enum Command {
 /// Tendermint consensus layer verifier.
 pub struct Verifier {
     protocol: Arc<Protocol>,
+    protocol_version: Version,
```

Comment:
I would rather call this `runtime_version` (as it is the version of the runtime binary).